scripts/ai/ProximityController.lua

```diff
@@ -65,6 +65,20 @@ function ProximityController:onBlockingVehicle(vehicle, isBack)
     end
 end
 
+--- Registers a function to ignore a object or vehicle.
+function ProximityController:registerIgnoreObjectCallback(object, callback)

```

Comment:
This does not appear to be used anywhere. Also, the sensor can already ignore vehicles, may consider using that mechanism.

Reply:
@pvaiko It's currently used to ignore the bunker silo walls for the proximitiy controller in the new strategy.
I will leave a TODO comment above the function and we consolidate this with vehicle at a later time?